docs/content/guides/6.multistore/2.tooling-and-concepts/4.deployment/7.configuration.md

```diff
@@ -7,11 +7,226 @@ navigation:
+
+::warning `region` Required
+Deployment cannot be triggered without a valid region configuration. The `region` value can be found in the [Deployment Variables](/console/instance/settings/deployment-variables) section of the Alokai Console. Ensure the value in `alokai.config.json` matches the region selected for your project.
+::
+
+Example region configuration:
+```json
+{
+  "stores": {
+    "fashion-brand": {
+      "deployment": {
+        "region": "us-east-1",  // AWS region identifier
+        // ... other parameters
+      }
+    }
+  }
+}
+```
+
+### `framework`
+
+::info Auto-detection
+The `framework` parameter is optional when your project uses only one frontend framework (Next.js or Nuxt). In this case, Alokai CLI will automatically detect and use the correct framework.
+::
+
+The `framework` parameter defines whether a store uses Next.js or Nuxt. Alokai supports both frameworks out of the box through two base applications:
+- `apps/storefront-unified-nextjs` for Next.js stores
+- `apps/storefront-unified-nuxt` for Nuxt stores
+
+Each store in `alokai.config.json` may specify which framework it will use for deployment:
+
+```json
+{
+  "stores": {
+    "fashion-brand": {
+      "deployment": {
+        "framework": "nextjs",  // This store will deploy using Next.js
+        // ... other parameters
+      }
+    },
+    "sports-brand": {
+      "deployment": {
+        "framework": "nuxt",  // This store will deploy using Nuxt
+        // ... other parameters
+      }
+    }
+  }
+}
+```
+
+Your project structure can support both frameworks simultaneously. For example:
+```bash
+apps/
+├── storefront-unified-nextjs/     # Base Next.js implementation
+├── storefront-unified-nuxt/       # Base Nuxt implementation
+└── stores/
+    ├── fashion-brand/            
+    │   ├── storefront-unified-nextjs/  # Next.js Storefront
+    └── sports-brand/
+        └── storefront-unified-nuxt/    # Nuxt Storefront
+```
+
+This dual-framework support is particularly beneficial when:
+- Different teams specialize in different frameworks
+- You're migrating from one framework to another
+- Specific stores have requirements better suited to a particular framework
+
+## Local Test Deployments
+
+::warning Development Environment Only
+Local deployments should only be used for development and testing purposes. For production deployments, we recommend using CI/CD pipelines (like GitHub Actions, GitLab CI, or Bitbucket Pipelines) as they provide better credentials' security, a controlled deployment environment, version control integration, and automated validation checks. You can read more about it in the [CI/CD guide](/guides/multistore/tooling-and-concepts/deployment/ci-cd).
+::
+
+### Prerequisites
+
+1. **Docker**
+   - Docker must be installed and running on your machine
+
+2. **Cloud Credentials**
+   - Find your credentials in the [Alokai Console](/console/instance/settings/deployment-variables)
+   - Use staging projects for local testing
```

Comment:
What is a "staging project"?

Reply:
oh, I was thinking about staging environment xd changed https://github.com/vuestorefront/vue-storefront/pull/7398/commits/feac843d800fec8f2eb12c935ff5fbd6514a83b9